dafni_cli/api/dafni_api.py

```diff
@@ -51,3 +51,27 @@ def dafni_post_request(
     )
     response.raise_for_status()
     return response.json()
+
+
+def dafni_put_request(
+    url: str, jwt: str, data: BinaryIO, content_type: str
```

Comment:
might be worth setting a default value?

Reply:
I'm not sure what a good default value would be. It's used twice in the upload, with two different values.
If when uploading datasets etc, there is a `content_type` which is used a lot, then I can add one then.

Reply:
ah yes, I did see that once I got quite far down. In that case leave as is